router/router_system_test.go

```
@@ -0,0 +1,55 @@
+	cfg.AuthenticationToken = "old-token"
+	if err := cfg.ResolveToken(false); err != nil {
+		t.Fatal(err)
+	}
+	config.Set(cfg)
+
+	credentials := make(chan [2]string, 1)
+	manager := server.NewEmptyManager(backupTestRemoteClient{credentials: credentials})
+	recorder := httptest.NewRecorder()
+	c, _ := gin.CreateTestContext(recorder)
+	c.Set("manager", manager)
+	c.Request = httptest.NewRequest("POST", "/api/update", strings.NewReader(`{"token_id":"new-id","token":"new-token"}`))
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)